content/tutorial/01-orders/01-basic-orders/04-submit/README.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Submitting
+---
+
+Further building on the previous tutorial, we will now submit the order we signed in the [sign order tutorial](/tutorial/sign) to CoW Protocol.
+
+## Submitting an order
+
+Submitting orders to the API may very well result in an error. For this reason, we should **ALWAYS** handle errors. To do this, we will use the `try/catch` syntax.
+
+```typescript
+/// file: run.ts
+import type { Web3Provider } from '@ethersproject/providers';
++++import { OrderBookApi, SupportedChainId, OrderQuoteRequest, OrderQuoteSideKindSell, OrderSigningUtils, UnsignedOrder, SigningScheme } from '@cowprotocol/cow-sdk';+++
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+
+    try {
+        const orderId = await orderBookApi.sendOrder({
+            ...quote,
+            ...orderSigningResult,
+            signingScheme: orderSigningResult.signingScheme as unknown as SigningScheme
+        })
+  
+        return { orderId }    
+    } catch (e) {
+        return e
+    }
+}
+```
+
+> Currently the `OrderSigningResult` returns an enum which is not compatible with the `SigningScheme` type. This is why we need to cast it to `unknown` and then to `SigningScheme`.
```

Comment:
Smells like a code smell 👃 

Is this something we should fix in the sdk?

Reply:
It would be great. Some differences have been located, and are tracked in https://github.com/cowprotocol/cow-sdk/issues/187